tools/wptserve/wptserve/response.py

```diff
@@ -183,6 +183,9 @@ def iter_content(self, read_file=False):
         True, the entire content of the file will be returned as a string facilitating
         non-streaming operations like template substitution.
         """
+        if not isinstance(self.content, Iterable) and not hasattr(self.content, "read"):
```

Comment:
I don't think the `Iterable` check works in general.

Reply:
No, it definitely doesn't. The vast majority of iterables don't descend from it.

Reply:
[Python documentation](https://docs.python.org/2.7/library/collections.html#collections.Iterable) says 
> `collections.Iterable`: ABC for classes that provide the `__iter__()` method. 

`isinstance(xxx, collections.Iterable)` returns `True` for all the sequence types and `dict` types and `file` objects. 
It's not the best way for checking iterable. Maybe I need to remove this `if` statement and add a `try...catch` statement whenever calling the `item_content` method.